packages/reassure-measure/src/types.ts

```diff
@@ -1,32 +1,32 @@
 /**
- * Type representing output from `measureRender` function.
+ * Type representing output of measure functions, e.g. `measurePerformance`, `measureFunction`.
  */
-export interface MeasureRenderResult {
-  /** Number of times the measurment test was run */
+export interface MeasureResults {
+  /** Number of times the test subject was run */
   runs: number;
 
-  /** Array of measured render durations for each run */
-  durations: number[];
-
-  /** Arithmetic average of measured render durations for each run */
+  /** Arithmetic average of measured execution durations for each run */
   meanDuration: number;
 
-  /* Standard deviation of measured render durations for each run */
+  /** Standard deviation of measured execution durations for each run */
   stdevDuration: number;
 
-  /** Array of measured render counts for each run */
-  counts: number[];
+  /** Array of measured execution durations for each run */
+  durations: number[];
 
-  /** Arithmetic average of measured render counts for each run */
+  /** Arithmetic average of measured execution count for each run */
   meanCount: number;
 
-  /** Standard deviation of measured render counts for each run */
+  /** Standard deviation of measured execution count for each run */
   stdevCount: number;
+
+  /** Array of measured execution count for each run */
+  counts: number[];
 }
 
 /**
  * Output of specific test scenarion as written to perf results file.
  */
-export interface PerformanceEntry extends MeasureRenderResult {
+export interface PerformanceEntry extends MeasureResults {
   name: string;
```

Comment:
```
type MeasureType = 'render' | 'syncFunction'
// ...

type: MeasureType;
```

pls add `type: 'render' | 'execution'`

Reply:
I'm thinking about the naming we can settle on here and I like your proposal with the `syncFunction` - especially if we want to support async executions in the future. Both _render_ and _syncFunction_ can _execute_, so maybe we can drop this.

I'll go with `'render' | 'syncFunction'` for now 👍🏼